src/test/regress/sql/shard_rebalancer.sql

```diff
@@ -1359,6 +1392,18 @@ FROM pg_dist_shard
 JOIN pg_dist_shard_placement USING (shardid)
 WHERE logicalrelid = 'r1'::regclass;
 
+SELECT replicate_reference_tables();
+
+-- After replication, the move should succeed.
+SELECT citus_move_shard_placement(pg_dist_shard.shardid, nodename, nodeport, 'localhost', :worker_2_port)
+    FROM pg_dist_shard JOIN pg_dist_shard_placement USING (shardid)
+    WHERE logicalrelid = 'd1'::regclass AND nodename = 'localhost' AND nodeport = :worker_1_port LIMIT 1;
+
+-- And a JOIN with the reference table should work.
+SELECT 1 as dummy FROM d1 JOIN r1 USING (a);
+
+DROP TABLE d1;
```

Comment:
This line should be deleted

Reply:
Are you sure? This cleans up the table I introduced earlier for this test (l. 1383).

Reply:
This would only be the case if you move the rest up I guess.